toast-tk-runtime/src/main/java/io/toast/tk/runtime/result/ObjectResultHandler.java

```diff
@@ -9,10 +14,36 @@
     public ObjectResultHandler(IActionItemRepository objectRepository) {
         super(objectRepository);
     }
-    
+
+	@SuppressWarnings({ "unchecked", "rawtypes" })
+    private String resultBuetifier(Object value) {
+    	String res = "";
+		
+		try {
+			res = (String) ((List)value).stream()
+					.map(Object::toString)
+					.collect(Collectors.joining(System.lineSeparator()));
+		}
+		catch(Exception e) {
+			try {
+				res = "";
+				for(Entry entry : ((Map<?, ?>) value).entrySet()) {
+					res += entry.getKey().toString() + " = " + entry.getValue().toString() + System.lineSeparator();
+				}
+			} catch(Exception e2) {
+				if(value != null) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2589)